pennylane/ops/op_math/condition.py

```diff
@@ -28,14 +28,14 @@ class ConditionalTransformError(ValueError):
     """Error for using qml.cond incorrectly"""
 
 
-class Conditional(SymbolicOp):
+class Conditional(SymbolicOp, Operation):

```

Comment:
Making `Conditional` also a subclass of `Operation` such that it inherits `grad_method`, which is required for the gradient transform.

Reply:
We should probably double check the status of parameter shift + conditionals.  If we had previously been supporting parameter shift + conditionals due to defer measurements being applied earlier, we may need some to think about what we want to do for it.

Reply:
What is the status here?

Reply:
Right now existing support is maintained.